Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:3.13-alpine
+FROM python:3.14.0rc3-alpine3.22
```

python3.14 is just a release candidate, i'd prefer setting 3.14.7
```suggestion
FROM python:3.13.7-alpine3.22
```